DemoApp/src/main.cc

```diff
@@ -1,34 +1,117 @@
 #include <iostream>
+#include <sstream>
 
 #include "Echo.h"
+#include "Utils.h"
 #include "views.h"
```

Comment:
to tak ma być, że wszystkie nazwy nagłówków zaczynają się wielką literą, a ten nie?

Reply:
No on ma tylko inne nagłówki w środku, w sumie nwm, klasa to to nie jest.

Reply:
No to nie jest klasa żadna, tylko zbiór nagłówków wszystkich Actions, nwm w sumie.